app/src/main/java/com/antosik/benchproject/app/popular/movies/viewModel/PopularMoviesViewModel.kt

```diff
@@ -38,23 +37,7 @@ class PopularMoviesViewModel @Inject constructor(
     private fun loadPopularMovie() {
         viewModelScope.launch {
             _isLoaderVisible.value = false
```

Comment:
Probably the order should be changed: 1. set the data 2. hide loader

Reply:
I will back to the earlier way with this loader because when I don't have any data in the database loader never stop :D

Reply:
I added TODO to app/src/main/java/com/antosik/benchproject/app/common/binding/view/ViewBindingAdapters.kt